libs/langchain-qdrant/src/vectorstores.ts

```diff
@@ -234,13 +233,15 @@ export class QdrantVectorStore extends VectorStore {
 
     await this.ensureCollection();
 
-    const results = await this.client.search(this.collectionName, {
```

Comment:
Updated the `similaritySearchVectorWithScore()` method to use the query API instead of the old search API.